### ISSUE TITLE
About Detection and REID

### ISSUE THREAD

**Issue by lichunjiang19960224** [kamkyu94/TrackTrack#16]
To ensure a fair comparison, can you provide a complete pickle file containing detection and features? Furthermore, I found that mot17_val_80 does not exist. Thank you.

**Comment by ZJZhao123** [comment]
> To ensure a fair comparison, can you provide a complete pickle file containing detection and features? Furthermore, I found that mot17_val_80 does not exist. Thank you.

我解析了一下mot17的95pickle文件，看到是跑了3段训练作为val，但是之前的ocsort之类的方法一般都是选择视频的前半段作为train后半段val，也有点奇怪

**Comment by ZJZhao123** [comment]
> To ensure a fair comparison, can you provide a complete pickle file containing detection and features? Furthermore, I found that mot17_val_80 does not exist. Thank you.

我自己生成了80的pickle文件。如果你需要的话可以联系我分享给你

**Comment by lichunjiang19960224** [comment]
> > To ensure a fair comparison, can you provide a complete pickle file containing detection and features? Furthermore, I found that mot17_val_80 does not exist. Thank you.
> 
> 我自己生成了80的pickle文件。如果你需要的话可以联系我分享给你

谢谢，我自己生成了，但是硬件不同，检测可能不一样。不太能控制变量。

**Comment by kamkyu94** [comment]
ZJZhao123 Thank you for sharing your effort!

Also, here is another answer.

"Many methods use the first half of the training videos for training and the remaining half for validation. In contrast, we construct custom validation sets for MOT17 and MOT20 by separating training sequences in a video-wise manner. (#training:#validation = 4:3 for MOT17, 2:2 for MOT20) The original way can be problematic since many objects in the training set also appear in the validation set. Our method prevents the duplication of objects between the training and validation sets."
